### PR TITLE
Firebase Crash Reporting for iOS

### PR DESCRIPTION
**Pull request by SotoiGhost**
This solves issue #56

### CODE REVIEW

File: Firebase.CrashReporting/component/Details.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+### **Important note:** *This component is only compatible with Xamarin Studio and Visual Studio for Mac.*
+
+Firebase Crash Reporting uses shell scripts to upload symbols to Firebase Console, but scripts use commands that are only available in Mac. Therefore, this component is only compatible with Xamarin Studio and Visual Studio for Mac.
+
+# Firebase Crash Reporting on iOS
+
```
Comment (SotoiGhost) [inline Firebase.CrashReporting/component/Details.md]: Any suggestion?

File: Firebase.CrashReporting/component/GettingStarted.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+### **Important note:** *This component is only compatible with Xamarin Studio and Visual Studio for Mac.*
+
+Firebase Crash Reporting uses shell scripts to upload symbols to Firebase Console, but scripts use commands that are only available in Mac. Therefore, this component is only compatible with Xamarin Studio and Visual Studio for Mac.
+
```
Comment (SotoiGhost) [inline Firebase.CrashReporting/component/GettingStarted.md]: Any suggestion?

**Comment by lekve11** [comment]
I can not add component in Xamarin Studio iOS

**Comment by alexrainman** [comment]
How to report a handled exception with this?

**Comment by SotoiGhost** [comment]
@alexrainman Firebase Crash Reporting for iOS is now deprecated, please, use Firebase Crashlytics for iOS instead.

**Comment by alexrainman** [comment]
I want to but Crashlytics and Performance Monitoring packages are missing for Android.

**Comment by alexrainman** [comment]
I am ok with that as soon as i have Android counterpart

**Pull request merged**